docs/gen-1/red-blue/main-glitchless/beginner-route/lass/README.md

```diff
@@ -1,15 +1,25 @@
 # Pokemon Red Glitchless Beginner Guide, Lass Version
 
+# Pokemon Red Glitchless Beginner Guide, No-Lass Version
+
 **FAQ: [Please read the FAQ before learning :)](../../resources/faq.md)**
 
+### Resources
+- [How To Speedrun Pokemon on Emulator](https://youtu.be/nf51DyXg-OY)
+- Watch the [RBY Manip Guide](https://youtu.be/QirPrbub21g?si=ec8acTpec_vi7moy)
+- [Resources - Pokémon Red/Blue](https://www.speedrun.com/pkmnredblue/resources)
+	- SAVES: Download "Any% Glitchless (Beginner Route)"
+ 	- RBY Savestate Patcher can be used to make your own practice ROM
+
 ### Manip quick reference
 - Nidoran Manip:
 	- [Optimal](../../resources/nido-manip.md) (HIGHLY RECOMMENDED)
-	- [1 A Press](https://youtu.be/2bwQZ6jPxRE)
+	- [Easy - 1 A Press](https://youtu.be/2bwQZ6jPxRE)
 - Mount Moon Manip:
-	- [Post Hiker Backup Paras](https://pastebin.com/j5gtY4cy) (recommended for first runs)
-		- [Older version](https://pastebin.com/7kcZu3g4) (easier path, but select yoloball)
+	- [Post Hiker Backup Paras](https://pastebin.com/j5gtY4cy) (recommended)
+		- [Easy - Older version](https://pastebin.com/7kcZu3g4) (easier path, but select yoloball)
 	- [Post Nerd Backup Paras](../../resources/postnerd-backup-paras.md)
+	- [Route 3 Moon Manip](https://pastebin.com/tggXpQRC)
```

Comment:
This was not listed on the lass document because its simply not usable with lass, i'm fine letting this here but hopefully it doesnt cause any confusion

Reply:
Oh that makes sense, I agree. Copy-paste error here on my part, will remove it.